### PR TITLE
Feature/details page clients

### PR DESCRIPTION
**Pull request by wilsoncooked**

### CODE REVIEW

File: src/components/global/BackButton.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function BackButton(props) {
+  const {buttonName} = props;
+  return (
+    <button type='button' className='btn'>
+      <i className='material-icons'>remove_circle</i>
```
Comment (laura-mso) [inline src/components/global/BackButton.js]:
```suggestion
      <i className='material-icons'>arrow_back</i>
```

File: src/components/client/ClientsDetail.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  componentDidMount() {
+    this.setState({id: this.props.id});
+    this.getClient();
+  }
+
+  render() {
+    if (this.state.clients === {}) {
+      return null;
+    }
```
Comment (laura-mso) [inline src/components/client/ClientsDetail.js]: I cant see why we need line 37 to 39

**Review by laura-mso — CHANGES_REQUESTED** [review]
Nice work!

File: src/components/client/ClientsDetail.js (hunk 2 above)
Comment (wilsoncooked) [reply src/components/client/ClientsDetail.js]: Neither was Jenna's code so maybe she needed it?

Comment (laura-mso) [reply src/components/client/ClientsDetail.js]: 🤣 I think you can remove it

File: src/components/global/BackButton.js (hunk 1 above)
Comment (wilsoncooked) [reply src/components/global/BackButton.js]: But this icon matches the edit button icon nicely

File: src/components/client/ClientsDetail.js (hunk 2 above)
Comment (wilsoncooked) [reply src/components/client/ClientsDetail.js]: Okay will do

Comment (wilsoncooked) [reply src/components/client/ClientsDetail.js]: Done!

**Review by laura-mso — APPROVED** [review]

**Review by aliashkevich — APPROVED** [review]

**Pull request merged**